src/web/rest_resources_viz/core.cljs

```diff
@@ -15,17 +15,15 @@
 (env/def
   RESOURCE_DATA_URL "graph-data.edn")
 
+(def node-colors ["#000000" "#0cc402" "#fc0a18" "#aea7a5" "#5dafbd" "#d99f07" "#11a5fe" "#037e43" "#ba4455" "#d10aff" "#9354a6" "#7b6d2b" "#08bbbb" "#95b42d" "#b54e04" "#ee74ff" "#2d7593" "#e19772" "#fa7fbe" "#62bd33" "#aea0db" "#905e76" "#92b27a" "#03c262" "#878aff" "#4a7662" "#ff6757" "#fe8504" "#9340e1" "#2a8602" "#07b6e5" "#d21170" "#526ab3" "#015eff" "#bb2ea7" "#09bf91" "#90624c" "#bba94a" "#a26c05"])
```

Comment:
I took them from a some website, they were ok, I tried to use d3 for generating them programmatically, but there was still something missing, so I tweaked them a bit.
I guess that instead of a list I can have a map from family to color, it'll read better. Any visual choice is of course negotiable :smile: